src/layout/workspace.rs

```diff
@@ -718,11 +718,17 @@ impl<W: LayoutElement> Workspace<W> {
         idx: usize,
         prev_idx: Option<usize>,
     ) -> f64 {
-        match self.options.center_focused_column {
-            CenterFocusedColumn::Always => {
+        match (
+            self.options.always_center_single_column,
+            self.options.center_focused_column,
+        ) {
+            (true, _) if self.columns.len() <= 1 => {
```

Comment:
I'd rather write it as a separate `if` at the front. `if self.options.always_center_single_column && self.columns.len() <= 1 { return ...; }`